src/views/datasets/booster-gap/dashboard.vue

```diff
@@ -0,0 +1,463 @@
+<template>
+  <DashboardCard width="full">
+    <template #subtitle>
+      This tool shows us places where fewer people have received a booster dose
+      compared to state levels. We call this difference in booster doses a
+      <em>gap</em>. Vaccines can keep us from getting very sick with COVID-19,
+      but protection decreases over time. A booster is another dose of the
+      vaccine that keeps your community protected. You can use this information
+      to find where the gaps are in our state and take steps to help close them.
+    </template>
+  </DashboardCard>
+
+  <DashboardCard width="full" :height="1">
+    <template #content>
+      <ControlPanel
+        :drop-downs="dropDowns"
+        :init-value="controls"
+        @selected="updateControls"
+      />
+    </template>
+  </DashboardCard>
+
+  <DashboardCard width="full" :height="5">
```

Comment:
Perhaps in another PR we can take this out into a smaller component to be used in both dashboards